testing/zerotier-one/APKBUILD

```diff
@@ -1,38 +1,51 @@
-# Contributor: Kyle Parisi <kyleparisi@gmail.com>
-# Maintainer: Kyle Parisi <kyleparisi@gmail.com>
+# Contributor: ZeroTier, Inc. <contact@zerotier.com>
+# Maintainer: Adam Ierymenko <adam.ierymenko@zerotier.com>
+
+## Thanks to Kyle Parisi <kyleparisi@gmail.com> for doing the original package and getting it into testing
+
+## Small changes by Leonardo Amaral <alpinelinux@leonardoamaral.com.br>
```

Comment:
Please, abuild is not an diary. There may be multiple `Contributor` lines, so just add the second one with your contact.
Also `Maintainer` and `Contributor` should be a real person, not a company.